funding/commitment_type_negotiation.go

```diff
@@ -278,13 +290,14 @@ func explicitNegotiateCommitmentType(channelType lnwire.ChannelType, local,
 		lnwire.ZeroConfRequired,
 	):
 
-		if !hasFeatures(
+		if ok, missings := hasFeatures(
```

Comment:
nit: s/missings/missing (for all the spots in this diff)

Reply:
Thanks for the catch